lua/nvim-treesitter/parsers.lua

```diff
@@ -968,6 +968,15 @@ list.kotlin = {
   maintainers = { "@SalBakraa" },
 }
 
+list.kusto = {
+  install_info = {
+    url = "https://github.com/Willem-J-an/tree-sitter-kusto",
+    files = { "src/parser.c" },
+  },
+  filetype = "kusto",
```

Comment:
not needed if it's not different from the grammar name

Reply:
Thanks, fixed